Signal/src/ViewControllers/HomeView/HomeViewCell.m

```diff
@@ -302,7 +313,7 @@ - (NSAttributedString *)attributedStringForDate:(nullable NSDate *)date
 
 - (UIFont *)dateTimeFont
 {
-    return [UIFont ows_dynamicTypeFootnoteFont];
+    return [UIFont ows_dynamicTypeFootnoteFont].ows_medium;
```

Comment:
nit: This might be better renamed to ows_mediumWeight - in context it's not clear what it's doing.

Reply:
Done.